cyprecice/cyprecice.pyx

```diff
@@ -568,6 +582,11 @@ cdef class Interface:
         >>> positions.shape
         (5, 3)
         """
+        check_array_like(vertex_ids, "vertex_ids", "get_mesh_vertices")
+
+        if not isinstance(vertex_ids, np.ndarray):
+            positions = np.asarray(vertex_ids)
```

Comment:
Is this necessary?

Reply:
Good eye! It's not. `np.ascontiguousarray` already takes care of converting everything that we get into a `np.ndarray`.

Reply:
Removed this line and tests still work. The test `test_set_mesh_vertex_list` should be testing exactly this case.

Reply:
Took care of this case an similar ones in c065016d0fe622e8b5c4de9754289316c7c046dd. 

Some insights: Converting to a numpy array is only neaded, of `input.shape` is required before converting `input` via `np.ascontiguousarray`. This is only the case for vector valued input (vector data and positions), where we want to make sure that number of dimensions and samples fit with the corresponding ids.